pact/coin-contract/coin-sig.pact

```diff
@@ -22,20 +22,28 @@
 
   (defun transfer-and-create:string (sender:string receiver:string receiver-guard:guard amount:decimal)
     @doc "Transfer between accounts SENDER and RECEIVER on the same chain.    \
-    \This fails if the SENDER account does not exist. If the RECEIVER account \
-    \does not exist, it is created and associated with GUARD."
+    \This fails if both accounts do not exist. Create-on-transfer can be      \
+    \done using the transfer-and-create function."
 
     @model [ (property (> amount 0.0))
              (property (not (= sender receiver)))
            ]
     )
 
   (defun account-balance:decimal (account:string)
-    @doc "Check an account's balance")
+    @doc "Check an account's balance"
+    @model [ (property (not (= amount ""))) ]
+    )
 
   (defun account-info:object (account:string)
-    @doc "Get all of an account's info.  This includes the balance and the    \
-    \guard.")
+    @doc "Get all of an account's info. This includes the balance and the     \
+    \guard."
+    @model [ (property (not (= amount ""))) ])
```

Comment:
Does this model make sense?  This function does not have an `amount` parameter.

Reply:
Thanks - it should be `account`